web_monitoring/tests/test_differs.py

```diff
@@ -67,15 +69,20 @@ def test_html_diff_render():
                   '<html><head><title>HTML Difference Render</title></head><body><h1>Head</h1></body></html>')]
 
-    test_results = ['<html>\n <style type="text/css">\n  ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n </style>\n <head></head>\n <body><h1><ins>Heading</ins></h1> <p><del>Paragraph</del></p></body>\n</html>',
-                    '<html>\n <style type="text/css">\n  ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n </style>\n <body><h1><ins>Heading</ins></h1> <p>Paragraph</p></body>\n</html>',
-                    '<html>\n <style type="text/css">\n  ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n </style>\n <body><h3>Paragraph</h3></body>\n</html>',
-                    '<html>\n <style type="text/css">\n  ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n </style>\n <head><title>HTML <ins>Difference</ins> <del>Diff</del> Render</title></head>\n <body><h1><ins>Head</ins></h1> <h1><del>Heading</del></h1></body>\n</html>']
-
-    test_check = []
+    test_results = ['<html>\n <head>\n  <style type="text/css">\n   ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n  </style>\n </head> <body>\n  <h1><ins>Heading</ins></h1> <p><del>Paragraph</del></p>\n </body>\n</html>',
+                    '<html>\n <head>\n  <style type="text/css">\n   ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n  </style>\n </head> <body>\n  <h1><ins>Heading</ins></h1> <p>Paragraph</p>\n </body>\n</html>',
+                    '<html>\n <head>\n  <style type="text/css">\n   ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n  </style>\n </head> <body>\n  <h3>Paragraph</h3>\n </body>\n</html>',
+                    '<html>\n <head>\n  <title>HTML <ins>Difference</ins> <del>Diff</del> Render</title>\n  <style type="text/css">\n   ins {text-decoration : none; background-color: #d4fcbc;}\n                        del {text-decoration : none; background-color: #fbb6c2;}\n  </style>\n </head> <body>\n  <h1><ins>Head</ins></h1> <h1><del>Heading</del></h1>\n </body>\n</html>']
 
     for index in range(len(test_data)):
-        test_check.append(wd.html_diff_render(test_data[index][0],
-                                              test_data[index][1]) == test_results[index])
+        diff = wd.html_diff_render(test_data[index][0], test_data[index][1])
+        assert (f'{index}: ' + test_results[index]) == (f'{index}: ' + diff)
```

Comment:
:+1: I was planning on circling back for this. Much easier to understand test failures this way. We might later go one step further and make the `test_results` into a pytest parameterization.

Reply:
> We might later go one step further and make the test_results into a pytest parameterization.

Faaaaaaaancy!